learn/features/randomness.md

```diff
@@ -0,0 +1,95 @@
+
+Moonbeam utilizes verifiable random functions (VRF) to generate randomness that can be verified on-chain. A VRF is a cryptographic function that takes some input and produces random values, along with a proof of authenticity that these random values were generated by the submitter. The proof can be verified by anyone to ensure the random values generated were calculated correctly.
+
+There are two available sources of randomness that provide random inputs based on block producers' VRF keys and past randomness results: [local VRF](#local-vrf) and [BABE epoch randomness](#babe-epoch-randomness). Local VRF is determined directly within Moonbeam using the collator of the block's VRF key and the last block's VRF output. On the other hand, [BABE](https://wiki.polkadot.network/docs/learn-consensus#block-production-babe){target=_blank} epoch randomness is based on all the VRF produced by the relay chain validators during a complete [epoch](https://wiki.polkadot.network/docs/glossary#epoch){target=_blank}.
+
+You can interact with and request on-chain randomness using the randomness precompile contract, a Solidity interface that enables smart contract developers to access the randomness functionality through the Ethereum API. For more information, please check out the [Interacting with the Randomness Precompile](/builders/pallets-precompiles/precompiles/randomness){target=_blank} guide. You can also take a look at the [The Randomness Pallet](/builders/pallets-precompiles/pallets/randomness){target=_blank}, which can be used to obtain current randomness requests and results.
+
+## General Definitions {: #general-definitions } 
+
+- **Epoch** - a time duration in the BABE protocol that is broken into smaller time slots. Slots are discrete units of time six seconds in length. On Polkadot, one epoch lasts approximately 2,400 slots or 4 hours. On Kusama, one epoch lasts approximately 600 slots or 1 hour.
+- **Deposit** - an amount of funds required to request random words. There is one deposit per request. Once the request has been fulfilled, the deposit will be returned to the account that requested the randomness
+- **Block expiration delay** - the number of blocks that must pass before a local VRF request expires and can be purged
+- **Epoch expiration delay** - the number of epochs that must pass before a BABE request expires and can be purged
+- **Minimum block delay** - the minimum number of blocks before a request can be fulfilled for local VRF requests. Set to {{ networks.moonbase.randomness.min_vrf_blocks_delay }} blocks
```

Comment:
I would not put a value (`set to 2 blocks`) here, as it might be different for production networks (and you don't put values in any of the other ones